test/index.test.js

```diff
@@ -205,5 +206,24 @@ describe('index.js', () => {
       expect(config.streams).to.have.lengthOf(2);
       expect(config.streams[1].stream).to.be.instanceOf(SentryStream);
     });
+
+    it('should have logstash stream on the default port', () => {
+      process.env.LOGSTASH_HOST = 'samplehost';
```

Comment:
You must reset the variable in the afterEach, otherwise if we reorder the tests they will break